.fourmolu_skip_files.txt

```diff
@@ -0,0 +1,19 @@
+Cabal-syntax/src/Distribution/Compat/NonEmptySet.hs
```

Comment:
What is the reason for these exemptions?

Reply:
This is an outdated file, you should check out the exemption list in the Makefile instead.